types/passport/index.d.ts

```diff
@@ -16,8 +16,11 @@ declare global {
     namespace Express {
         // tslint:disable-next-line:no-empty-interface
         interface AuthInfo {}
-        // tslint:disable-next-line:no-empty-interface
-        interface User {}
+
+        interface User {
+            accessToken?: string;
```

Comment:
I can't find `accessToken` or `refreshToken` in `passport`, so this shouldn't get defined in `@types/passport`.

https://github.com/jaredhanson/passport/search?q=accesstoken&unscoped_q=accessToken